Sampler/Assets/RequiredModules.psd1.template

```diff
@@ -27,6 +27,8 @@
     'DscResource.Test'          = 'latest'
     'DscResource.AnalyzerRules' = 'latest'
     xDscResourceDesigner        = 'latest'
+    'DscResource.DocGenerator'  = 'latest'
```

Comment:
technically, this one should/could also be conditional of the Sampler module_type...

Reply:
It is already, several of the modules above is within an if block for module type 'dsccommunity'. :)